packages/patternfly-4/react-core/src/components/LoginPage/LoginBoxHeader.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Title } from '../Title';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly-next/components/LoginBox/login-box.css';
+import spacing from '@patternfly/patternfly-next/utilities/Spacing/spacing.css';
+import displayStyles from '@patternfly/patternfly-next/utilities/Display/display.css';
+import flexAlignment from '@patternfly/patternfly-next/utilities/Flex/flex.css';
+
+const propTypes = {
+  /** Content rendered inside the LoginBox Header */
```

Comment:
I wonder if there are any components here that are always required  for a loginPage Header.
@christiemolloy do you know?

Reply:
looping in @andybraren and @mcarrano to take a look at this

Reply:
I believe all of these would be required for any LoginPage Header. Display and Flex certainly, Spacing probably unless the dev wanted to make something super custom.

(Caveat: I’m not at all familiar with how devs use React to build things, I could be wrong.)

Reply:
So everything is required for the Header here?